pkg/config/config.go

```diff
@@ -44,9 +44,15 @@ func CheckAndLoad() error {
 	defer f.Close()
 
 	err = json.NewDecoder(f).Decode(&cfg)
+	if err != nil {
```

Comment:
Not sure if this will work if starting with an empty file. I have to check it.

Reply:
I hadn't thought of that. Maybe we should find a way to handle that case more explicitly?

Reply:
> Maybe we should find a way to handle that case more explicitly?

Yep.. I believe that's better.

Reply:
I think this is fixed in 538f9003f03cd1d1ce32665eb4090a1a9ecd3e38.